src/app/holdingpen-editor/holdingpen-save-button/holdingpen-save-button.component.ts

```diff
@@ -96,9 +96,9 @@ export class HoldingpenSaveButtonComponent extends SubscriberComponent implement
       .do(() => this.domUtilsService.unregisterBeforeUnloadPrompt())
       .subscribe(() => {
         window.open(`/holdingpen/${this.workflowObject}`);
-      }, (error: ApiError) => {
-        if (error.status === 404) {
-          this.jsonBeingEdited$.next(error.body);
+      }, (error: ApiError<WorkflowSaveErrorBody>) => {
+        if (error.status === 404 && error.body.error_code === 'WORKFLOW_INVALID') {
```

Comment:
😕  shouldn't be ```404 || error.body.error_code === 'WORKFLOW_INVALID'```

Reply:
no, I think idea is have the `error_code`s for only `status: 400`

(I'll fix the `status` though)